examples/pnp_demo/src/main.rs

```diff
@@ -107,26 +113,35 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         &rerun::Points2D::new(img_obs).with_colors([[0, 255, 0]]), // green
     )?;
 
-    // Reproject world points with estimated pose
-    let r_est = result.rotation;
-    let t_est = result.translation;
-    let mut img_reproj = Vec::with_capacity(world_pts.len());
-    for p in &world_pts {
-        let x_c = r_est[0][0] * p[0] + r_est[0][1] * p[1] + r_est[0][2] * p[2] + t_est[0];
-        let y_c = r_est[1][0] * p[0] + r_est[1][1] * p[1] + r_est[1][2] * p[2] + t_est[1];
-        let z_c = r_est[2][0] * p[0] + r_est[2][1] * p[1] + r_est[2][2] * p[2] + t_est[2];
-        let u = k[0][0] * x_c / z_c + k[0][2];
-        let v = k[1][1] * y_c / z_c + k[1][2];
-        img_reproj.push((u, v));
-    }
+    // Helper to reproject with a given pose
+    let reproject = |r: [[f32; 3]; 3], t: [f32; 3]| -> Vec<(f32, f32)> {
```

Comment:
i advocate to create later a project_points thing similar to opencv, using glam to make use of ismd instructions